addons/monitoring-prometheus-addon/src/main/java/org/kie/addons/monitoring/system/metrics/DMNResultMetricsBuilder.java

```diff
@@ -0,0 +1,31 @@
+package org.kie.addons.monitoring.system.metrics;
+
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+
+import io.prometheus.client.Counter;
+import org.kie.dmn.api.core.DMNDecisionResult;
+import org.kie.kogito.dmn.rest.DMNResult;
+
+public class DMNResultMetricsBuilder {
+
+    private static final ConcurrentHashMap<CountersTypesEnum, Counter> counters = new ConcurrentHashMap<>();
+
+    public static Counter GetCounter(){
```

Comment:
```suggestion
    public static Counter getCounter(){
```

please use lower case method names even for statics

Reply:
Thx for the catch! Resolved